requirements.in

```diff
@@ -13,6 +13,19 @@ requests-mock
 freezegun
 tabulate
 awslogs
+aws-cdk.core
+aws-cdk.aws_s3
+aws-cdk.aws_lambda
+aws-cdk.aws_dynamodb
+aws-cdk.aws_apigateway
+aws-cdk.aws_events_targets==1.32.0
```

Comment:
Because if you pin nothing, there will be conflicts because some modules depend on older versions of other modules. So you sometimes need to pin the modules that are dependencies to older versions.